common/kvstore/src/main/java/org/apache/spark/util/kvstore/LevelDB.java

```diff
@@ -76,7 +76,7 @@ public LevelDB(File path, KVStoreSerializer serializer) throws Exception {
     this.types = new ConcurrentHashMap<>();
 
     Options options = new Options();
-    options.createIfMissing(!path.exists());
+    options.createIfMissing(true);
```

Comment:
Tests generally use a temp dir for the db, using `Utils.createTempDir` or something like that which creates the directory for you. That would cause this to fail unless you deleted the directory first (which `LevelDBSuite` does), which I found a little bit annoying after a while.